src/utils.cpp

```diff
@@ -90,198 +82,200 @@ double pyne::endftod_cpp(char * s) {
       exp = s[10] + 10 * s[9] - 11 * '0';
-      dbl_exp = exp & 01? 10.: 1;
-      dbl_exp *= (exp >>= 1) & 01? 100.: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e4: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e8: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e16: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e32: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e64: 1;
-      dbl_exp = (s[8] == '-'? 1/dbl_exp: dbl_exp) * 1.0e-5;
-      v = mant * (s[0] == '-'? -1: 1) * dbl_exp;
+      dbl_exp = exp & 01 ? 10. : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 100. : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e4 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e8 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e16 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e32 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e64 : 1;
+      dbl_exp = (s[8] == '-' ? 1 / dbl_exp : dbl_exp) * 1.0e-5;
+      v = mant * (s[0] == '-' ? -1 : 1) * dbl_exp;
     }
   }
 
   // Convert an ENDF int to float; we start from the last char in the field and
   // move forward until we hit a non-digit.
   else {
     v = 0;
-    mant = 1; // Here we use mant for the place value about to be read in.
+    mant = 1;  // Here we use mant for the place value about to be read in.
     pos = 10;
     while (s[pos] != '-' and s[pos] != '+' and s[pos] != ' ' and pos > 0) {
       v += mant * (s[pos] - '0');
       mant *= 10;
       pos--;
     }
-    v *= (s[pos] == '-'? -1: 1);
+    v *= (s[pos] == '-' ? -1 : 1);
   }
   return v;
 }
 
-double pyne::endftod_f(char * s) {
+double pyne::endftod_f(char *s) {
 #ifdef PYNE_IS_AMALGAMATED
   return endftod_cpp(s);
 #else
   return endftod_(s, 12);
 #endif
 }
 
-double (*pyne::endftod)(char * s) = &pyne::endftod_f;
+double (*pyne::endftod)(char *s) = &pyne::endftod_f;
 
-void pyne::use_fast_endftod() {
-  pyne::endftod = &pyne::endftod_cpp;
-}
+void pyne::use_fast_endftod() { pyne::endftod = &pyne::endftod_cpp; }
 
 std::string pyne::to_upper(std::string s) {
   // change each element of the string to upper case.
-  for(unsigned int i = 0; i < s.length(); i++)
-    s[i] = toupper(s[i]);
+  for (unsigned int i = 0; i < s.length(); i++) s[i] = toupper(s[i]);
   return s;
 }
 
 std::string pyne::to_lower(std::string s) {
   // change each element of the string to lower case
-  for(unsigned int i = 0; i < s.length(); i++)
-    s[i] = tolower(s[i]);
+  for (unsigned int i = 0; i < s.length(); i++) s[i] = tolower(s[i]);
   return s;
 }
 
-
 std::string pyne::capitalize(std::string s) {
   unsigned int slen = s.length();
-  if (slen == 0)
-    return s;
+  if (slen == 0) return s;
   // uppercase the first character
   s[0] = toupper(s[0]);
   // change each subsequent element of the string to lower case
-  for(unsigned int i = 1; i < slen; i++)
-    s[i] = tolower(s[i]);
+  for (unsigned int i = 1; i < slen; i++) s[i] = tolower(s[i]);
   return s;
 }
 
+std::ostringstream pyne::line_wrapping(std::string line,
+                                       int line_lenght) {
```

Comment:
```suggestion
                                       int line_length) {
```
and matching changes throughout